packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -1,38 +1,43 @@
 {
   "extends": "./configs/base.json",
   "rules": {
-    "no-var": "error",
-    "prefer-const": "error",
-    "prefer-rest-params": "error",
-    "prefer-spread": "error",
     "@typescript-eslint/adjacent-overload-signatures": "error",
-    "@typescript-eslint/array-type": "error",
```

Comment:
This rule is definitely a "style" rule, but did you consider modifying it to

```
"@typescript-eslint/array-type": ["error", "array-simple"],
```

This version enforces `string[]` and `Array<{ /*object properties* }>`, while banning `Array<string>` or `{ /* object properties */ }[]`.  It's still a style opinion, but maybe it's a less onerous one, since this seems to be the style that most codebases gravitate to anyway.  (The `[]` at the end of long object definitions is IMO, pretty bad readability)

Reply:
we could potentially, but I want to tend away from style rules that are in regions that are opinionated.

I would rather have a slightly looser subset that keeps everyone happy, than have a stricter set which makes people want to spam us with issues/comments because they don't agree with it (which has happened. Like a lot, the opinionated minority can be _very_ vocal).